website/content/docs/other-specifications/namespace.mdx

```diff
@@ -108,20 +108,22 @@ consul {
 
 ### `node_pool_config` Parameters <EnterpriseAlert inline />
 
-- `default` `(string: "default")` - Specifies the node pool to use for jobs in
-  this namespace that don't define a node pool in their specification.
+- `default` `(string: "default")` - Specifies the node pool to use for jobs or
+  dynamic host volumes in this namespace that don't define a node pool in their
+  specification.
 
 - `allowed` `(array<string>: nil)` - Specifies the node pools that are allowed
-  to be used by jobs in this namespace. By default, all node pools are allowed.
-  If an empty list is provided only the namespace's default node pool is
-  allowed. This field supports wildcard globbing through the use of `*` for
-  multi-character matching. This field cannot be used with `denied`.
+  to be used by jobs or dynamic host volumes in this namespace. By default, all
```

Comment:
GH won't let me suggest changes to both L115 and 116 in a single comment (grrr).

Suggest:
Specifies the node pools that jobs or dynamic host volumes are allowed to use in this namespace.

to replace the "are allowed to be used" construct